gulpfile.js

```diff
@@ -290,13 +290,32 @@ function compile(watch, shouldMinify, opt_preventRemoveAndMakeDir,
 function compileCss() {
   $$.util.log('Recompiling CSS.');
   return jsifyCssAsync('css/amp.css').then(function(css) {
-    return gulp.src('css/**.css')
-        .pipe($$.file('css.js', 'export const cssText = ' + css))
-        .pipe(gulp.dest('build'));
+    var resolve;
+    var promise = Promise(r => {
+      resolve = r;
+    });
+    gulp.src('css/**.css')
+        .pipe($$.file('css.js', 'export const cssText = ' +
+            JSON.stringify(css)))
+        .pipe(gulp.dest('build'))
+        .on('end', function() {
+          fs.writeFileSync('build/amp.css');
+          resolve(css);
+        });
+    return promise;
   });
 }
 
 /**
+ * Copies the css from the build folder to the dist folder
+ */
+function copyCss() {
+  fs.copySync('build/amp.css', 'dist/amp.css');
+  return gulp.src('build/**/amp-*.css')
```

Comment:
currently I am. better all of them though right for consistency? or is it just the main binary that we need?